src/components/DatePicker/index.ios.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import {Button, View} from 'react-native';
+import DatePicker from '@react-native-community/datetimepicker';
+import moment from 'moment';
+import ExpensiTextInput from '../ExpensiTextInput';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import Popover from '../Popover';
+import CONST from '../../CONST';
+import styles from '../../styles/styles';
+import colors from '../../styles/colors';
```

Comment:
Should use theme colors
```suggestion
import themeColors from '../../styles/themes/default';
```